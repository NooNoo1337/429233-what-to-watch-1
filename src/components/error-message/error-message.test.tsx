import * as React from 'react';
import * as renderer from 'react-test-renderer';
// Components
import ErrorMessage from './error-message';

describe(`ErrorMessageComponent`, () => {
  it(`should render correctly`, () => {
    const tree = renderer
      .create(
        <ErrorMessage/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
