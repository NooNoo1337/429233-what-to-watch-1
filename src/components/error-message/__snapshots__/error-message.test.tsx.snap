// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessageComponent should render correctly 1`] = `
<div
  className="error-message"
>
  <h1
    className="error-message__text"
  >
    Ops! We got troubles with server. I'll fix it, promise.
  </h1>
</div>
`;
